articles/how-to-implement-mlops/index.md

```diff
@@ -0,0 +1,55 @@
+
+### What is MLOps
+
+The idea of MLOps was born out of the need to combine the long-established practice of DevOps with the recently emerging field of machine learning. MLOps automates the processes of [model development](https://www.sciencedirect.com/topics/computer-science/model-development#:~:text=Model%20development%20is%20an%20iterative,rather%20than%20where%20it%20finished.), drift monitoring, quality control, [model governance](https://algorithmia.com/blog/model-governance), [model retraining](https://docs.aws.amazon.com/machine-learning/latest/dg/retraining-models-on-new-data.html), and [pipelining](https://www.sciencedirect.com/topics/computer-science/pipelining) into a single platform.
+
+MLOps is modeled on the existing DevOps discipline, which involves developing and operating large-scale software systems. DevOps ensures the collaboration of software developers (the Devs) and operations teams (the Ops).
+
+Read more about DevOps engineering [here](https://www.section.io/engineering-education/what-it-takes-to-be-a-devops-engineer/).
+
+MLOps is a [compound](https://caiomsouza.medium.com/mlops-machine-learning-and-operations-and-ai-at-scale-ffcac7e50f62) of machine learning (ML) and operations (Ops). With this practice, operations professionals and data scientists collaborate and communicate better to manage [production machine learning](https://towardsdatascience.com/production-machine-learning-isnt-hard-anymore-932bd91e138f) or [machine learning life cycle](https://www.educba.com/machine-learning-life-cycle/#:~:text=Machine%20Learning%20Life%20Cycle%20is,are%20involved%20in%20various%20applications). Essentially, MLOps comes in to manage the deployment of deep learning and machine learning models in large-scale production environments.
+
+MLOps includes a data science team for curating datasets and building AI models that analyze the datasets. Machine learning engineers are also part of the MLOps team, and they have the role of running datasets through the models in automated ways.
+
+There exist three levels of MLOps, as [described](https://cloud.google.com/solutions/machine-learning/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning) by Google: MLOps level 0, MLOps level 1, and MLOps level 2. The choice between each level is dependent on an organization&#39;s size and the number of machine learning algorithms they target to run.
+
+- **MLOps level 0 -** This means thatan organization recognizes the need to adopt machine learning practices and have an in-house data scientist capable of deploying and building models. However, ML workflow in that entity remains mainly manual.
```

Comment:
The last sentence isn't clear. ML workflow in that entity? what do you mean by that?